### PR TITLE
IP & Progress

### PR DESCRIPTION
**Pull request by BoxedBrain**
as discussed

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -861,7 +861,7 @@ curl --output db.tar.gz https://iota.lukaseder.de/downloads/db.tar.gz
 ```
 Unpack it:
 ```sh
-tar zxvf db.tar.gz
+mkdir -p db && pv db.tar.gz | tar xzf - -C ./
```
Comment (nuriel77) [inline docs/README.md:864]: Actually, I think this file README.md can be removed. I started using readthedocs. 

Here's that page:
http://iri-playbook.readthedocs.io/en/docs/faq.html#where-can-i-get-a-fully-synced-database-to-help-kick-start-my-node

The file can be edited in the 'docs' branch (just branch off it) https://github.com/nuriel77/iri-playbook/blob/docs/docs/faq.rst

File: fullnode_install.sh [hunk 2]
```diff
@@ -123,8 +123,8 @@ function init_centos(){
         exit 0
     fi
 
-    echo "Installing Ansible, net-tools and git..."
-    yum install ansible git net-tools -y
+    echo "Installing Ansible, git and pv..."
+    yum install ansible git pv -y
```
Comment (nuriel77) [inline fullnode_install.sh:127]: Actually, here we're only installing what is required to run the playbook.
All other dependencies are installed in the playbook. There are users who don't use this script. They clone and run the playbook. That's why its better to add it here for yum:
https://github.com/nuriel77/iri-playbook/blob/master/roles/iri/tasks/deps_yum.yml#L42

Comment (BoxedBrain) [reply fullnode_install.sh:127]: Okay, added "pv" also to the playbook.

File: docs/README.md (hunk 1 above)
Comment (BoxedBrain) [reply docs/README.md:864]: Going to edit "docs" branch. Are you going to merge docs into master?

**Pull request closed without merging**